src/createAdder.js

```diff
@@ -20,7 +20,15 @@
  * @param {number} initialValue
  */
 function createAdder(initialValue = 0) {
-  // write code here
+  let accumulator = initialValue;
+
+  return (...arg) => {
+    [ ...arg ].forEach(argument => {
```

Comment:
скобки не нужны, фор ич не трогает оригинальный массив
не arg а args, множественное число же
```suggestion
    args.forEach(argument => {
```